### ISSUE TITLE
Windows 10 1909 to 2004 upgrade issue & 2004 issue

### ISSUE THREAD

**Issue by kwgagel** [DavidXanatos/DiskCryptor#12]
Windows would not upgrade with DiskCryptor 1.1 installed. Once un-installed the upgrade proceeded.
Re-installing v1.1 on Win 10 ver 2004 went OK but when I launch DiskCryptor 1.1 it just want's to reboot the comptuer with this message:
![image](https://user-images.githubusercontent.com/20076204/95141271-8c106080-0725-11eb-866d-873328bade56.png)
I upgraded DiskCryptor to 1.2 beta and it is doing the same message. No amount of restarts removes the message and DiskCryptor remains unusable.

**Comment by DavidXanatos** [comment]
is the driver missing or somehow blocked?
try disabling secure boot in bios, you will need to do that to use the bootloader anyway

**Comment by kwgagel** [comment]
> is the driver missing or somehow blocked?
> try disabling secure boot in bios, you will need to do that to use the bootloader anyway

Sorry, I can't find anything in my BIOS labeled as "Secure Boot". I can say that the driver isn't installing. I tried reinstalling the app and get this error message.
![image](https://user-images.githubusercontent.com/20076204/95396241-58644080-08b5-11eb-9f5b-5286cfd20658.png)

**Comment by kwgagel** [comment]
Any updates, comments, suggestions, directions??

**Comment by DavidXanatos** [comment]
> 
> 
> Any updates, comments, suggestions, directions??

Yes: https://www.geoffchappell.com/notes/security/whqlsettings/index.htm
this article describes a couple of registry key that can be used to disable the WHQL requirement on secure boot systems.

Alternatively you can get a driver signed with an old certificate that works just fine but is often flagged by anti malware fools.

**Comment by kwgagel** [comment]
I tried the registry settings and still couldn't make it work. I rechecked my BIOS and turned off Intel's trusted platform and it still didn't install. How do I get the older signed driver?

**Comment by DavidXanatos** [comment]
Older driver? you mean from the no UEFI compatible releases? that wont help eider.
You could try Build v1.2 Beta 2 it uses a different code signing certificate, one that many anti malware fools complain about.
But its one that at least untill recently was accepted even by secure boot systems

**Comment by kwgagel** [comment]
I'm getting this error with Build 1.2 Beta 3. I have downloaded build 1.2 beta 2 but when I unzip it I get this error:
![image](https://user-images.githubusercontent.com/20076204/96506326-4a46e600-120c-11eb-93a6-d26fdc8dd528.png)

**Comment by kwgagel** [comment]
That doesn't work either. I get the same error trying to install Beta 2 version.

**Comment by kwgagel** [comment]
Any other suggestions? I have an encrypted database I need to access and can't because the driver won't install...

**Comment by dcherubini** [comment]
I had the same issue after upgrading to Windows 20H2. I had uninstalled DiskCryptor before updating Windows, and installed it again after the update was complete. After that DiskCryptor was asking me to reboot the computer every time I launched it.

What worked for me was simply uninstalling DiskCryptor again (it showed an error but finished the process successfully), then re-installing it and rebooting one more time. After that it started working just fine.
